ush/run_fv3.ksh

```diff
@@ -0,0 +1,39 @@
+#!/bin/ksh --login
+ 
+# Set the queueing options 
+#SBATCH --nodes=60 --ntasks-per-node=4   ## refer to the run_fcst task section in the workflow xml
+#SBATCH -t 6:40:00
+#SBATCH -A nrtrr
+# #SBATCH -qos debug
+#SBATCH -J fv3_fcst
+#SBATCH --partition=sjet  ## check which jet partition was used from the run_fcst log file in the cycle that needs to be repeated
+ 
+set -x
+#-------------------------------------------
+# point to the source code for executable and
+# for loading the same modules used in the run that needs to be repeated
+#-------------------------------------------
+SRCDIR=/WHERE-YOUR-ufs-srweather-app-CODE-IS/dev4-ufs-srweather-app
+module purge
+module use $SRCDIR/regional_workflow/modulefiles/tasks/jet
+module load run_fcst
+ 
+FV3_EXE=$SRCDIR/exec/fv3_gfs.x
+
+#-------------------------------------------
+# the workdir is a hard copy of the cycle that needs to be repeated, for example, /lfs4/BMC/nrtrr/NCO_dirs/stmp/tmpnwprd/RRFS_dev4/2021010612
+# remove the IC/LBC source (HRRRX/, RAPX/), forecast files (dynf*.nc, phyf*.nc), log files (logf*), post and restart files (if applicable) 
+# must keep all the fix files and INPUT/ 
+#-------------------------------------------
+workdir=/WEHER-YOUR-TEST-DIR-IS/2021010612
```

Comment:
Typo in "WHERE"

Reply:
Corrected. Thanks for noticing it!

Reply:
And yes it is a good idea to discuss with Joe and others to see what they think.

Reply:
I'd prefer it if nothing was cleaned/purged by this script. Let it age off the disk after ~15 hrs and the entire directory can be wiped at that point. That's how the HRRR-system was generally configured.

Reply:
@joeolson42, this script is not meant to ever be run in real time. This is meant to be the reproducibility helper script you requested yesterday that is just the job card for submitting the executable for a given run directory -- the same as the workflow would have done.

Reply:
OK... I'm a bit confused because I was thinking that this script would be **exactly equivalent** to what was used in realtime, so we can immediately reproduce the realtime crashes. But maybe this is close enough?

Reply:
The purpose of this script is to reproduce the run cycle in the real-time in users' space, without the complex feature of running the workflow. Just like in WRF, we can copy over wrfprd/ and run the stand-alone script if we have the run script. This script works similarly. After users copy over the FV3 run directory (where you have the INPUT/ and get the forecast run) to their local space, they can use this script to mimic the environment as used in the real-time (queue options, module files, fix files, source code and executable) and repeat the forecast job in specified users' space. I have tested it and it produce identical forecasts.

Reply:
I think we're on the same page